tokio-threadpool/src/pool/mod.rs

```diff
@@ -78,6 +79,8 @@ pub(crate) struct Pool {
 
     // Configuration
     pub config: Config,
+
+    span: tokio_trace::Span,
```

Comment:
Would be good to have a comment above this field since every other field has it :)